test/test_passive.js

```diff
@@ -0,0 +1,121 @@
+const {eg_config, parser_tests} = require('./utils.js');
```

Comment:
Really minor but can we just stick to `require('./utils')` without the .js everywhere.